tools/msg_allgather.py

```diff
@@ -0,0 +1,18 @@
+import numpy as np
+import torch
+import torch.distributed as dist
+
+def executeAllGather( sendData, worldSize ): 
```

Comment:
please remove the space before and after the arguments.

Reply:
Also we use `snake_case` everywhere in Python code except for class names.